src/GloryDuckTables2019Lkl.cc

```diff
@@ -232,7 +236,11 @@ Int_t GloryDuckTables2019Lkl::ReadGloryDuckInputData(TString filename)
     {
       if(read_logJ)
         {
-          logJ = field;
+          if(TMath::Abs(field-fLogJ) > 1.e-6)
```

Comment:
Agreed (see my answer above) but I didn't touch it for now. Just to be sure: in that case you would compute the lkl values that are correct for the J-factor of the rcfile, ie the one we want to use for combination, using the J-factor written in the file as "start point" (because it is the one used to compute the lkl actually stored in the file).

Reply:
Yes, that is what I mean. In this way you are allowed to compute <sv> values for any J-factor, and not just the one written in the file

Reply:
Done.